src/Fabulous.XamarinForms/Xamarin.Forms.Core.Attributes.fs

```diff
@@ -6,6 +6,15 @@ open Xamarin.Forms
 
 module Application =
     let MainPage = Attributes.defineWidget ViewNode.getViewNode "Application_MainPage" (fun target -> (target :?> Xamarin.Forms.Application).MainPage) (fun target value -> (target :?> Xamarin.Forms.Application).MainPage <- unbox value)
+    let Resources = Attributes.defineWidget ViewNode.getViewNode "Application_Resources" (fun target -> (target :?> Xamarin.Forms.Application).Resources) (fun target value -> (target :?> Xamarin.Forms.Application).Resources <- unbox value)
```

Comment:
Resources need to be declared the same way than https://github.com/TimLariviere/Fabulous-new/pull/13 until we figure out #10 
Right now, my guess is that it will crash if you try to use it

Reply:
I think it won't crash is we pass a resource dictionary and also define Resources as a scalar attribute for now . until we create  Resources as a widget . I have added a example on the CounterApp